main/models/category.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, ForeignKey, NVARCHAR, Text
+
+from main.models.base_class import Base
+
+
+class Category(Base):
```

Comment:
Should be `CategoryModel` as we might have several entities for the category such as CategorySchema, CategoryService, etc.